docs/advanced/graphql/dataloaders-and-cache-invalidation.mdx

```diff
@@ -468,9 +468,18 @@ export default {
       supports: "*",
       config: {
         // see https://www.npmjs.com/package/redis#options-object-properties
-        host: "127.0.0.1",
-        port: 6379,
-        db: 1,
+        host:
+          process.env.FRONT_COMMERCE_CLOUD_REDIS_HOST || // Front-Commerce Cloud variables
```

Comment:
instead of mixing the documentation of the cache with the cloud documentation, wouldn't it make sense to add a page dedicated to the cloud setup ?